pynamodb/models.py

```diff
@@ -45,9 +45,10 @@ class ModelContextManager(object):
 
     """
 
-    def __init__(self, model, auto_commit=True):
+    def __init__(self, model, auto_commit=True, delay=0):
```

Comment:
You add it to the base class but it should only apply to a specific subclass of `BatchWrite`?